src/openvic-simulation/country/Country.hpp

```diff
@@ -0,0 +1,124 @@
+#pragma once
+
+#include <filesystem>
+#include <memory>
+#include <string>
+#include <string_view>
+#include <type_traits>
+#include <unordered_map>
+#include <vector>
+
+#include <openvic-dataloader/v2script/AbstractSyntaxTree.hpp>
+
+#include "openvic-simulation/politics/Ideology.hpp"
+#include "openvic-simulation/pop/Culture.hpp"
+#include "openvic-simulation/types/Colour.hpp"
+#include "openvic-simulation/types/Date.hpp"
+#include "openvic-simulation/types/Dictionary.hpp"
+#include "openvic-simulation/types/IdentifierRegistry.hpp"
+
+namespace OpenVic {
+	struct GameManager;
+	struct CountryManager;
+
+	struct CountryParty {
+		friend struct CountryManager;
+
+	private:
+		const std::string name;
+		const Date start_date;
+		const Date end_date;
+		const Ideology& ideology;
+		const std::string economic_policy;
+		const std::string trade_policy;
+		const std::string religious_policy;
+		const std::string citizenship_policy;
+		const std::string war_policy;
```

Comment:
These can be looked up and stored as pointers or references to the registered instance, similar to `ideology`.